template.rb

```diff
@@ -104,7 +104,17 @@ def add_template_repository_to_source_path
 
 say "=> Copying git configuration"
 directory "lib/install/.github", ".github"
-copy_file "lib/install/.gitattributes", ".gitattributes"
+gitattributes = <<-CODE
+  *.gemspec diff=ruby
+  *.rake    diff=ruby
+  *.rb      diff=ruby
+  *.js      diff=javascript
+
+  db/schema.rb merge=ours diff=ruby
+  yarn.lock merge=ours
+  Gemfile.lock merge=ours linguist-generated
+CODE
+insert_into_file '.gitattributes', gitattributes
```

Comment:
but what to do for rails 6.0? there are no such file/

Reply:
Updated